test.diff

```diff
@@ -0,0 +1,241 @@
+diff --git a/src/s3.c b/src/s3.c
```

Comment:
Probably you didn't mean to add in this file.  Please remove it from your pull request.